packages/larva-css/src/03-utilities/u-width.common.inline.scss

```diff
@@ -1,17 +1,25 @@
 @import 'setup';
 
 $widths_map: (
+	'width-16': pmc-rem(16),
```

Comment:
@yahilmadakiya after this is merged, can you please go through AMH and replace the local utilities with the larva utils?

Reply:
Sure,

Reply:
Sure